### PR TITLE
Phase 1 Stargate Celebrate Stakin

### PR DESCRIPTION
**Pull request by EdouardLvdl**
Blog post will be updated within 48 hours to reflect latest changes and updates 🌳

### CODE REVIEW

File: aplikigo-1/tasks/phase-1/1-celebrate/stakin.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "moniker": "Stakin",
+    "valopr_address": "regen:valoper1cf79sumhf35ayq8n2e0szy266gsmf8rv054rvj",
+    "sendTxHashes": ["EDF6769C4BFFC65EECB323189BF4291A764577058952DFEFF894A6BB1AB0EC05", "A8DCF7056902D2E2D7F7222DA315AED8C96B2B49F269973265B86B1E1A1F6171"],
+    "delegateTxHashes": ["B4886AD12C5B19203D9D963F6E5D1E57FD052426FAF87BB736CD7DC1B5F6611D", "5216A1FA8EA336AFC62329B93DF53FBA8C2D1A2A258657E153CB364E99AFA41B"],
+    "tweetMemoTx": "https://twitter.com/StakinOfficial/status/1358803823826001921?s=20",
+    "linkedInPostMemoTx": "https://www.linkedin.com/feed/update/urn:li:activity:6764569905208545280",
+    "memoWithDiscordHandleTxHash": "AD9C370F5CEBA7508C85214786CE9310551CF6B6E890B514A55FDBA8D2498D76",
```
Comment (aditya-manit) [inline aplikigo-1/tasks/phase-1/1-celebrate/stakin.json:8]: What command did you use?
regen tx data sign [signer] [cid] [flags] --> whats cid here?

Comment (EdouardLvdl) [reply aplikigo-1/tasks/phase-1/1-celebrate/stakin.json:8]: Thanks for reviewing the PR @aditya-manit 

What transaction are you referring to please?

Are you referring the memoDiscordHandleTxHash? I used a bank tx with Discord memo

What's [cid] here please?

Comment (EdouardLvdl) [reply aplikigo-1/tasks/phase-1/1-celebrate/stakin.json:8]: Thanks for reviewing the PR @aditya-manit 

What transaction are you referring to please?

Are you referring the memoDiscordHandleTxHash? I used a bank tx with Discord memo

What's [cid] here please?

Comment (aditya-manit) [reply aplikigo-1/tasks/phase-1/1-celebrate/stakin.json:8]: Hey, I am not from the team, saw your PR thought you would know and could help me.
"I used a bank tx with Discord memo" --> yeah everyone is using this command apparentlya and not this one "regen tx data sign [signer] [cid] [flags] "

Btw just to let know you're too early, did you read this?

Time: Feb 9th, 1500UTC - Feb 11th, 1200 UTC (Only the txs in this time period are considered to be valid)
here: https://github.com/regen-network/testnets/blob/master/aplikigo-1/PLAN.md

Comment (EdouardLvdl) [reply aplikigo-1/tasks/phase-1/1-celebrate/stakin.json:8]: Thanks for the help. I m not familiar with data tx but will try it. 

Didn't pay attention to the dates. I'll amend the PR, thanks again!

Comment (aditya-manit) [reply aplikigo-1/tasks/phase-1/1-celebrate/stakin.json:8]: Np :)

**Pull request closed without merging**